components/blobserve/pkg/blobserve/blobserve.go

```diff
@@ -102,7 +106,19 @@ func (reg *Server) MustServe() {
 // serve serves a single file from an image
 func (reg *Server) serve(w http.ResponseWriter, req *http.Request) {
 	vars := mux.Vars(req)
-	repo, tag := vars["repo"], vars["tag"]
+	image := vars["image"]
+	pref, err := reference.ParseNormalizedNamed(image)
```

Comment:
I don't think we want to support the Docker shortcut here, but rather stick to OCI distribution naming which mandates that registries have a domain.

Reply:
I'm afraid I don't understand what you're trying to tell me. :laughing: What is your suggestion?

Reply:
Do you propose to use `reference.ParseNamed(image)` instead of `reference.ParseNormalizedNamed(image)` so that we accept `myreg.com/alpine:1.2.3` but not `alpine:1.2.3`?